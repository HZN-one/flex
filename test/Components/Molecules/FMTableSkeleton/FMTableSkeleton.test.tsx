/**
 * @jest-environment jsdom
 */

import * as React from "react";
import "@testing-library/jest-dom";
import { FMTableSkeleton } from "@Molecules";

import { render } from "@testing-library/react";

describe("FMTableSkeleton", () => {
  it("should match snapshot", () => {
    const { container } = render(
      <FMTableSkeleton columns={5} rows={2} testID="table-skeleton-example" />
    );
    expect(container).toMatchSnapshot();
  });
});
