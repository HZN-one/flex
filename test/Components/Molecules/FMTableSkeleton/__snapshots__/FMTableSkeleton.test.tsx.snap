// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FMTableSkeleton should match snapshot 1`] = `
<table>
  <tbody
    class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
    data-testid="table-skeleton-example"
  >
    <tr
      class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
    >
      <td
        class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
      >
        <span
          class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-wave css-1dk1h2u-MuiSkeleton-root"
          style="height: 20px;"
        />
      </td>
      <td
        class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
      >
        <span
          class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-wave css-1dk1h2u-MuiSkeleton-root"
          style="height: 20px;"
        />
      </td>
      <td
        class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
      >
        <span
          class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-wave css-1dk1h2u-MuiSkeleton-root"
          style="height: 20px;"
        />
      </td>
      <td
        class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
      >
        <span
          class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-wave css-1dk1h2u-MuiSkeleton-root"
          style="height: 20px;"
        />
      </td>
      <td
        class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
      >
        <span
          class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-wave css-1dk1h2u-MuiSkeleton-root"
          style="height: 20px;"
        />
      </td>
    </tr>
    <tr
      class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
    >
      <td
        class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
      >
        <span
          class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-wave css-1dk1h2u-MuiSkeleton-root"
          style="height: 20px;"
        />
      </td>
      <td
        class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
      >
        <span
          class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-wave css-1dk1h2u-MuiSkeleton-root"
          style="height: 20px;"
        />
      </td>
      <td
        class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
      >
        <span
          class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-wave css-1dk1h2u-MuiSkeleton-root"
          style="height: 20px;"
        />
      </td>
      <td
        class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
      >
        <span
          class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-wave css-1dk1h2u-MuiSkeleton-root"
          style="height: 20px;"
        />
      </td>
      <td
        class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
      >
        <span
          class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-wave css-1dk1h2u-MuiSkeleton-root"
          style="height: 20px;"
        />
      </td>
    </tr>
  </tbody>
</table>
`;
